src/server/index.jsx

```diff
@@ -327,9 +351,16 @@ server
         } else {
           throw new Error('unknown result');
         }
-      } catch ({ message, status }) {
+      } catch ({ message, status = 500 }) {
+        await sendCustomMetric({
+          metricName: NON_200_RESPONSE,
+          statusCode: status,
```

Comment:
Is this combined with L354 potentially misleading, as we can't distinguish between an Ares 500 and one we have defaulted to 500?

Reply:
I think it can be misleading yes, however this is currently how it works. So perhaps this needs to be looked into further?